app/controllers/api/v0/versions_controller.rb

```diff
@@ -118,6 +155,7 @@ def serialize_version(version, options = {})
     methods << :change_from_earliest if boolean_param(:include_change_from_earliest)
     options[:methods] = methods
 
+    version.uri = "#{api_v0_version_url(version)}/raw"
```

Comment:
Should be `raw_api_v0_version_url(version)`, no string substitution or manual path writing necessary.

I would also feel a *lot* better if we didn’t mutate and dirty the actual model object here (otherwise, once you’ve called this method, you’d better be careful not to do anything that might cause it to save and you also can’t read the `uri` field anymore). Instead, we should modify the hash returned from `as_json`.

Reply:
Ah, thank you for this. This felt like an icky choice when I did it, but I wasn't sure what the correct choice was.

Reply:
`rake routes` is a very useful tool if you aren’t totally up on how Rails names these things :)

Reply:
Yeah, I used `rake routes` while setting up my initial PR for this issue, but it seemingly fell out of my brain by the time I came around to this new approach. 😳This should be in a happier place now.